doc/doc-site/docs/off-chain/rust-sdk.md

```diff
@@ -1,3 +1,255 @@
+
+# Structure
+
+- The main struct of the SDK is `EconiaClient`. This allows you to submit transactions, query events and get a view client (`EconiaViewClient`). To create a new `EconiaClient`, you need a node URL, the Econia contract's address, an `aptos_sdk::types::LocalAccount` and you can optionally pass a client config.
+
+- In `econia_sdk::entry`, you can find helper functions to create transactions for every entry function of the Econia contract. These can then be submitted using the `EconiaClient::submit_tx`.
+
+- To access view functions, you can use `EconiaClient::view_client`, which will return an `EconiaViewClient`, which has bindings for every view function in the Econia client.
+
+# Example script
+
+You can see the source of the script [here]().
+
+## Setup
+
+To run the example script, you first need to run a local Aptos node:
+
+```bash
+brew install aptos # only if necessary
+mkdir aptos && cd aptos
+aptos node run-local-testnet --with-faucet
+```
+
+In another terminal, run the following:
+
+```bash
+export APTOS_NODE_URL=http://0.0.0.0:8080
+export APTOS_FAUCET_URL=http://0.0.0.0:8081
+```
+
+You'll also have the option of entering these as prompts of the script, but the environment variable is preferred because it's easier to run multiple times. It's time to deploy our own Econia Faucet to the local chain:
+
+```bash
+git clone https://github.com/econia-labs/econia.git # only if necessary
+cd ./econia/src/move/faucet
+aptos init --profile econia_faucet_deploy # enter "local" for the chain
+export FAUCET_ADDR=<ACCOUNT-FROM-ABOVE>
+# deploy the faucet (all one command)
+aptos move publish \
+        --named-addresses econia_faucet=$FAUCET_ADDR \
+        --profile econia_faucet_deploy \
+        --assume-yes
+```
+
+You also need to deploy Econia:
+
+```bash
+cd ./econia/src/move/econia
+aptos init --profile econia_exchange_deploy # enter "local" for the chain
+export ECONIA_ADDR=<ACCOUNT-FROM-ABOVE>
+# deploy the exchange (all one command)
+aptos move publish \
+        --override-size-check \
+        --included-artifacts none \
+        --named-addresses econia=$ECONIA_ADDR \
+        --profile econia_exchange_deploy \
+        --assume-yes
+```
+
+It's time to run the script! Setting our environment variables will have cleared the initial setup prompts for us. In order to run, go into the `econia/src/rust/sdk/example` folder and run `cargo run -- $APTOS_NODE_URL $APTOS_FAUCET_URL $ECONIA_ADDR $FAUCET_ADDR`.
+
+## Understanding the example script
+
+The script is separated in steps. To go to the next step, press enter.
+
+### Initialization
+
+Before the first step, there is an initialization phase where the Rust code creates the necessary variables for the rest of the code, and creates and funds a main account. This is handled by the `init` helper function. The `account` helper function creates a new Aptos account and funds it through the faucet.
+
+```rust
```

Comment:
As we discussed on the eng call today, let's take these out to avoid having to maintain duplicate code snippets between source/docs

If someone wants to harvest code they can always look through the script, so I think for these sections it'll be more appropriate to give a general walkthrough of what the script does here, then provide the terminal feedback 

(See https://econia.dev/off-chain/python-sdk#understanding-the-example-script)

Reply:
Again, this applies to multiple sections below but I won't replicate the comment for each instance